src/Symfony/Component/Form/Extension/Core/DataTransformer/PercentToLocalizedStringTransformer.php

```diff
@@ -34,16 +34,19 @@ class PercentToLocalizedStringTransformer implements DataTransformerInterface
     private $roundingMode;
     private $type;
     private $scale;
+    private $html5Format;
 
     /**
      * @see self::$types for a list of supported types
      *
-     * @param int    $scale The scale
-     * @param string $type  One of the supported types
+     * @param int      $scale        The scale
+     * @param string   $type         One of the supported types
+     * @param int|null $roundingMode a value from \NumberFormatter, such as \NumberFormatter::ROUND_HALFUP
+     * @param bool     $html5Format  Use a HTML5 specific format, as per https://www.w3.org/TR/html51/sec-forms.html#date-time-and-number-formats
      *
      * @throws UnexpectedTypeException if the given value of type is unknown
      */
-    public function __construct(int $scale = null, string $type = null, ?int $roundingMode = null)
+    public function __construct(int $scale = null, string $type = null, ?int $roundingMode = null, bool $html5Format = false)
```

Comment:
I think I would rather allow passing a `$locale` and `$grouping` argument like in the `NumberToLocalizedStringTransformer`.

Reply:
I have missed feelings about that: HTML5 format happens to be `en` locale without grouping, but it's accidental. I feel like it is correct to expose clearly that we want HTML5-compatible format in the normalizer interface.